packages/react-core/src/components/InputGroup/examples/InputGroup.md

```diff
@@ -32,6 +32,7 @@ import QuestionCircleIcon from '@patternfly/react-icons/dist/esm/icons/question-
 
 ### With popover
 
+using `appendTo` with ref to the `InputGroup` for the Popover so that the popover width will depend on the whole input group
```

Comment:
For the propComponents array at the top of the file, we just need to add in InputGroupItem:

```
propComponents: ['InputGroup', 'InputGroupItem', 'InputGroupText']
```